### PR TITLE
Rename prepare_transaction() to prepare_send_outputs()

### PR DESCRIPTION
**Pull request by Thoralf-M**
Rename so it's more aligned with our other methods

### CODE REVIEW

File: sdk/src/wallet/operations/transaction/prepare_send_outputs.rs
Comment (thibault-martinez) [inline sdk/src/wallet/operations/transaction/prepare_send_outputs.rs:1]: can we rename this file to `send_outputs` and have both `send_outputs` and `prepare_send_outputs` in it pls? More consistent with the rest

Comment (Thoralf-M) [reply sdk/src/wallet/operations/transaction/prepare_send_outputs.rs:1]: Done

**Review by thibault-martinez — DISMISSED** [review]

**Review by DaughterOfMars — COMMENTED** [review]
Not sure I like this one...I mean you're not sending outputs really?

**Comment by Thoralf-M** [comment]
> Not sure I like this one...I mean you're not sending outputs really?

How is this different to sending accounts/NFTs?

**Comment by DaughterOfMars** [comment]
Well in those cases you are transferring ownership of the IDs, but that's not true with outputs.

**Review by DaughterOfMars — DISMISSED** [review]

**Review by DaughterOfMars — APPROVED** [review]

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**